src/index.html

```diff
@@ -6,13 +6,58 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__img">
+        <img
+          class="card__item"
+          src="./images/imac.jpeg"
+          alt="imac"
+        />
+      </div>
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <h3 class="card__code">Product code: 195434</h3>
+
+      <div>
+        <ul class="stars stars--4">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+      </div>
+
+      <div class="card__price">Price:</div>
+
+      <div class="card__buy">
+        <a
+          href="#"
+          class="card__bottom"
+          data-qa="hover"
+        >
+          Buy
+        </a>
+      </div>
```

Comment:
The class `card__bottom` does not follow BEM methodology. It should be `card__button` or `card__link` to clearly represent the element. Also, the text 'Buy' should be in a `span` tag with a BEM class like `card__button-text`.